interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -142,6 +143,11 @@ namespace cling {
     ///
     CompilationOptions m_Opts;
 
+    ///\brief If DefinitionShadower is enabled, the `__cling_N5xxx' namespace
+    /// in which to nest global definitions (if any).
+    ///
+    clang::NamespaceDecl* m_DefinitionNS;
```

Comment:
Could you add `Shadow` to the member name? Could you add in-class init `= nullptr`?

Reply:
Sure. :-)